src/queue-client.js

```diff
@@ -0,0 +1,79 @@
+const QUEUE_ATTRIBUTES_FALLBACK = 'Queue attributes unavailable.';
+const jobNotFound = id => `Job ${id} not found.`;
+
+class QueueClient {
+  constructor(bullQueue, queueTimeout = 5000) {
```

Comment:
We use 20 seconds for long polling with the SQS, not sure if we want to mimic that or not since this is our own queue on our own resources. Set to 5 secs to be pragmatic.